Fluid/ConcurrentMemberAccessStrategy.cs

```diff
@@ -19,18 +19,36 @@ public IMemberAccessor GetAccessor(Type type, string name)
         {
             IMemberAccessor accessor = null;
 
-            while (type != typeof(object))
+            var currentType = type;
+
+            while (currentType != typeof(object) && currentType != null)
             {
                 // Look for specific property map
-                if (_map.TryGetValue(type, out var typeMap))
+                if (_map.TryGetValue(currentType, out var typeMap))
                 {
                     if (typeMap.TryGetValue(name, out accessor) || typeMap.TryGetValue("*", out accessor))
                     {
                         return accessor;
                     }
                 }
 
-                type = type.GetTypeInfo().BaseType;
+                currentType = currentType.GetTypeInfo().BaseType;
+            }
+
+            // Search for accessors defined on interfaces
+            foreach (var interfaceType in type.GetTypeInfo().GetInterfaces())
```

Comment:
I refactored this part to prevent the recursion. The end result is the same but the execution is faster.

Reply:
I guess I have to dig into the documentation again, the recursion was to support interfaces that are implemented by base classes as well.

Reply:
I think the interfaces are for the whole hierarchy. Can you check?

Reply:
Yes, checked that, you are right.

Reply:
https://gist.github.com/SebastianStehle/6402c465941662cd275a7bcd5da532e3